enterprise/server/oidc/oidc.go

```diff
@@ -303,11 +302,11 @@ func newOpenIDAuthenticator(ctx context.Context, env environment.Env, oauthProvi
 	}
 
 	claimsFunc := claims.ParseClaims
-	if *claimsCacheTTL > 0 {
-		claimsCache, err := claims.NewClaimsCache(ctx, *claimsCacheTTL)
-		if err != nil {
-			return nil, err
-		}
+	claimsCache, err := claims.NewClaimsCache(ctx)
```

Comment:
seems fine but in general if err is nil it's assumed that the value is safe to use, and in this case it's nil, so that's a little error prone. This is the only callsite I see, but can you make sure to add a comment on func NewClaimsCache that this may return nil?

Reply:
Done